external/common/requirements.txt

```diff
@@ -13,9 +13,9 @@ unittest2==0.5.1
 validictory==0.9.1
 PyMySQL==0.6.2
 DBUtils==1.1
-pyproj==1.9.3
+pyproj>=1.9.3
```

Comment:
@scottpurdy cc @oxtopus pyproj 1.9.5 was added to pypi today, and as can be seen on AppVeyor cannot be built from source. I'd recommend changing my change here back to `pyproj==1.9.3`